docs/designs/sanitize-html.md

```diff
@@ -0,0 +1,144 @@
+# Sanitize HTML
+
+[Feature 178608](https://dev.azure.com/ceapex/Engineering/_workitems/edit/178608/)
+
+## Overview
+
+Content authors can embed arbitrary HTML tags to markdown documents. These HTML tags could contain `<script>` tag that allows arbitrary script execution, or `style` attribute that compromises site visual experience.
+
+### Goals
+
+- Remove HTML tags and attributes from user content that may alter site style
+- Remove HTML tags and attributes from user content that may allow javascript execution
+- Apply HTML sanitization consistently for all user content
+
+### Out of scope
+
+- Report warnings for removing disallowed HTML tags and attributes
```

Comment:
Why is this out of scope? Perf? 
1. It would be awesome if these violations were visible to authors.
1. It would be super duper awesome if these violations went into some reporting sink so we could identify and chip away at content using unsupported markup.

From the template/ux perspective we really don't want folks using html at all.

Reply:
It was in a separate feature. But it make sense to include both in this spec.